inst/private/python_ipc_system.m

```diff
@@ -113,12 +113,21 @@
         disp (['This session will use the temp file "' tmpfilename '"'])
       end
     end
-    fd = fopen (tmpfilename, 'w');
+    [fd, msg] = fopen (tmpfilename, 'w');
+    if fd < 0
+      error (msg)
+    end
     fprintf(fd, '# temporary autogenerated code\n\n');
     % we just added two more lines at the top
     info.prelines = info.prelines + 2;
-    fputs(fd, bigs);
-    fclose(fd);
+    status = fputs(fd, bigs);
+    if status < 0
+      error ('failed to write the big string to temporary file')
+    end
+    status = fclose(fd);
+    if status < 0
+      error ('failed to close temporary file after writing')
+    end
```

Comment:
Similarly, we can prepend `system ipc:` to the error message and show the temp file fd here (if needed).